.github/workflows/ci.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
+    - name: GitHub Action for npx
+      uses: mikeal/npx@1.0.0
```

Comment:
Is this step necessary? `npx` should be able to run within Github Actions without this. We run `npx` in other Github Action workflow files within other repos without this step.

Reply:
Not necessary, removed.